src/table/table.scss

```diff
@@ -122,265 +236,315 @@
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
-
-      > .iui-full-line {
-        flex: 1;
-      }
-
-      > .iui-controls {
+      &::selection {
         @include themed {
-          color: t(iui-text-color);
-        }
-        width: 300px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex: 1;
-
-        > a {
-          @include iui-anchor;
-          margin: 0 $iui-s;
-        }
-
-        > .iui-ellipsis {
-          margin: 0 $iui-s;
-        }
-
-        > .iui-current,
-        > .iui-ellipsis {
-          @include themed {
-            color: t(iui-text-color);
-          }
-          cursor: default;
+          background-color: rgba(
+            t(iui-color-foreground-warning-rgb),
+            t(iui-opacity-4)
+          );
         }
+      }
+    }
+  }
 
-        > .iui-current {
-          text-decoration: underline;
-          font-weight: $iui-font-weight-semibold;
-        }
+  &.iui-negative {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-negative-rgb),
+        t(iui-opacity-6)
+      );
+    }
 
-        > a > .iui-icon {
-          @include themed {
-            color: t(iui-icons-color-actionable);
-            fill: t(iui-icons-color-actionable);
-          }
-          display: block;
-          width: $iui-icons-default;
-          height: $iui-icons-default;
+    &::selection {
+      @include themed {
+        background-color: rgba(
+          t(iui-color-foreground-negative-rgb),
+          t(iui-opacity-4)
+        );
+      }
+    }
+  }
 
-          cursor: pointer;
+  &.iui-editable {
+    cursor: pointer;
+    @include iui-focus;
 
-          &:hover {
-            @include themed {
-              color: t(iui-icons-color-actionable-hover);
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+    &:hover {
+      @include themed {
+        box-shadow: inset 0 0 0 1px t(iui-color-foreground-primary);
       }
+    }
+  }
+}
 
-      > .iui-location {
-        display: flex;
-        justify-content: flex-end;
-        align-items: center;
-        flex: 1;
-
-        > .iui-input-container {
-          margin: 0;
+@mixin iui-table-row-status($status: positive) {
+  &,
+  + .iui-expanded-content {
+    @include themed {
+      box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-#{$status});
+    }
 
-          > .iui-select {
-            @include themed {
-              border-color: t(iui-color-background-3);
-              background-color: t(iui-color-background-3);
-              color: t(iui-text-color-muted);
-            }
-            cursor: pointer;
-            padding: 0 $iui-baseline 0 $iui-baseline/2;
-            height: auto;
-
-            &:hover,
-            &:focus {
-              @include themed {
-                background-color: none;
-              }
-            }
-          }
-        }
+    *::selection {
```

Comment:
Sure thats the expected selection color on non-status cell (in statused row) ?
![image](https://user-images.githubusercontent.com/81580355/120783761-07841500-c534-11eb-8107-18c595366c9c.png)